README.md

```diff
@@ -66,17 +72,22 @@ flowchart TD
 
 ![check-python-version](docs/check-python-version.gif)
 
-----
+[[back to top](#table-of-contents)]
 
-# How to Use Transpile
+----
 
-## Step 1 : Installation
+# Installation
 
 Upon completing the environment setup, install Remorph by executing the following command:
 ```bash
 databricks labs install remorph
 ```
+[[back to top](#table-of-contents)]
 
+----
+# How to Use Transpile
+
+## Step 1 : Verify Installation
 Verify the successful installation by executing the provided command; confirmation of a successful installation is indicated when the displayed output aligns with the example screenshot provided:
 ```bash
  databricks labs remorph transpile --help
```

Comment:
Add help for recon as well, since install can install for both `transpile` as well as `reconcile`

Reply:
It is already there .. I added it in `How to Use Reconcile` part